src/common/select/Select.tsx

```diff
@@ -1,47 +1,45 @@
-import React from 'react';
-import classNames from 'classnames';
-
-import styles from './select.module.scss';
+import React, { ChangeEvent } from 'react';
+import { Dropdown } from 'hds-react';
 
 interface Option {
   label: string;
   value: string | number;
```

Comment:
> value: string | number;

API suggestion: could this value be made generic? This would ofcourse also need a converter from the generic type to string, which could simply default to `() => String(value)` if none is provided.